package.json

```diff
@@ -1,34 +1,13 @@
 {
   "name": "wallabag",
-  "version": "2.1.0",
+  "version": "2.2.2",
```

Comment:
The `version` field is required, but it doesn't really matter what we put in it.